soql-server-pg/src/test/resources/fixtures/select-curated-region-test-invalid-geometry.json

```diff
@@ -1,2 +1,2 @@
-[{"name":"Chili", "test_result": "Self-intersection[1 1]" }
```

Comment:
I have no idea why this changed!  But I ran the query that generates it over in the SoQLGeomFunctionTest and indeed it returns 2 1, not 1 1?  So was, somehow, the resource name's optionalness masking a bug before?

Reply:
...and on jenkins it returns the original value??? I am so confused.